add info dir section and entry to texdraw and epspdf
---
---
 texmf/doc/info/epspdf.info	2012-03-06 11:43:52.019809190 +0900 |binary
 texmf/doc/info/epspdf.info  |    5 +++++
 texmf/doc/info/texdraw.info |    5 +++++
 2 files changed, 10 insertions(+)

Index: texlive-base-2011.20120226/texmf/doc/info/texdraw.info
===================================================================
--- texlive-base-2011.20120226.orig/texmf/doc/info/texdraw.info	2006-01-03 09:00:18.000000000 +0900
+++ texlive-base-2011.20120226/texmf/doc/info/texdraw.info	2012-03-06 11:43:12.000000000 +0900
@@ -14,6 +14,11 @@
 entire resulting derived work is distributed under the terms of a
 permission notice identical to this one.
 
+INFO-DIR-SECTION TeX
+START-INFO-DIR-ENTRY
+* TeXdraw: (texdraw).      A system for producing PostScript drawings from TeX.
+END-INFO-DIR-ENTRY
+
 
 File: texdraw,  Node: Top,  Next: Introduction,  Prev: (dir),  Up: (dir)
 
Index: texlive-base-2011.20120226/texmf/doc/info/epspdf.info
===================================================================
--- texlive-base-2011.20120226.orig/texmf/doc/info/epspdf.info	2011-02-27 22:21:24.000000000 +0900
+++ texlive-base-2011.20120226/texmf/doc/info/epspdf.info	2012-03-06 11:43:52.019809190 +0900
@@ -9,6 +9,11 @@
      the copyright notice and this notice are preserved.  This file is
      offered as-is, without any warranty.
 
+INFO-DIR-SECTION TeX
+START-INFO-DIR-ENTRY
+* epspdf: (epspdf).           Epspdf and epsdftk.
+END-INFO-DIR-ENTRY
+
 
 File: epspdf.info,  Node: Top,  Next: Usage,  Prev: (dir),  Up: (dir)
 
